apps/tenant-management-webapp-e2e/src/integration/comment.feature

```diff
@@ -0,0 +1,50 @@
+@comment
+Feature: Comment
+
+  @TEST_CS-2435 @REQ_CS-2399 @regression
+  Scenario: As a tenant admin, I can see an overview of the comment service, so I understand what it is used for
+    Given a tenant admin user is on comment service overview page
+    Then the user views the "Comment service" overview content "Comment services allows"
+    And the user views the link of API docs for "Comment service"
+    And the user views the link of See the code for "comment-service"
+    And the user views the link of "Get support" under Support
+
+  @TEST_CS-2441 @TEST_CS-2440 @TEST_CS-2442 @TEST_CS-2445 @TEST_CS-2524 @REQ_CS-2393 @REQ_CS-2395 @REQ_CS-22396 @REQ_CS-2394 @REQ_CS-2448 @regression
+  Scenario: As a tenant admin, I can add, edit and delete a comment topic type with data classification
+    Given a tenant admin user is on comment service overview page
+    When the user clicks Add topic type button on comment service overview page
+    Then the user views Add topic type modal
+    When the user clicks Cancel button in Add topic type modal
+    Then the user views topic types page
+    # Invalid data
+    When the user clicks Add topic type button on topic types page
+    Then the user views Add topic type modal
+    When the user enters "auto-test-1-$" in Add topic type modal
+    Then the user views the error message of "Allowed characters are: a-z, A-Z, 0-9, -, [space]" for Name field in Add topic type modal
+    # Validate data
+    When the user enters "autotest-topicType" in Add topic type modal
+    And the user clicks Save button in Add topic type modal
+    Then the user views topic type editor for "autotest-topicType"
+    And the user views "Protected A" as default selection for security classification
+    And the user views "Public, Protected A, Protected B, Protected C" in Select a security classification dropdown
+    When the user enters "empty" as classification, "auto-test-role1" as admin roles, "auto-test-role2" as commenter roles, "empty" as reader roles
```

Comment:
I don't see the option to select as "empty" as Select a security classification, it should be as one of the options

Reply:
Change "empty" to "skip" after disucssion.